pom.xml

```
@@ -10,7 +10,7 @@
 	</parent>
 	<artifactId>openbis-client-lib</artifactId>
 	<packaging>jar</packaging>
-	<version>1.4.1</version>
+	<version>1.4.2</version>
```

Due to added features, according to semantic versioning this must be
```suggestion
	<version>1.5.0</version>
```

---

This is no longer a hotfix

---

base changed to dev branch, version to 1.5.0 snap